code/index.html

```diff
@@ -1,30 +1,181 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- bootstrap css -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <!--google fonts link-->
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lexend+Giga&family=Noto+Sans+JP:wght@300&family=Oswald&family=Roboto:wght@300&display=swap"
+    rel="stylesheet" />
+  <!--bootstrap js-->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+  <!-- main css link-->
+  <link rel="stylesheet" href="./style.css">
+  <title>Coffee shop</title>
+</head>
+
+</body>
+
+<header>
+  <!--video-header for DESKTOP-->
+<div class="header-text">
+  <h1>Coffee!</h1>
+    <span>is always a good Idea</span>
+</h1>
+    </div>
+  <div class="video-container">
+    <video playsinline autoplay muted loop poster="placeholder.jpg" width="100%" height="800">
+      <source src="https://res.cloudinary.com/dgujdyxlv/video/upload/v1599342050/video_resized_header_zxwgok.mp4" type="video/mp4">
+    </video>
+  </div>
```

Comment:
Remember to check your indentation and code structure, this is SO important and will make things easier for yourself!